README.md

```diff
@@ -7,25 +7,45 @@
 
 ## Group Members
 
+1. Brent
+2. Coulter
+3. Jackson
+
 
 ## Project Description
 
+Building a web-app for users to connect with developers and pair program
+
 
 ## Who uses it?
 
+We will have 2 types of users. Students & Teachers
 
 ## What outputs do they need?
 
+Being connected to a chat app
+
```

Comment:
This isn't an output- what do users expect to get when they use this app. Chat, sure. But think in terms of "coding instruction delivered in real time over a chat app" "access to the screen of their instructor/student counterpart" etc. There are a lot of outputs a user would expect here- tell us all of them